src/dev/impl/DevToys/ViewModels/Tools/Text/StringUtilities/StringUtilitiesToolViewModel.cs

```diff
@@ -626,6 +634,182 @@ private void ExecuteInverseCaseCommand()
 
         #endregion
 
+        #region AlphabetizeCommand
+
+        public IRelayCommand AlphabetizeCommand { get; }
+
+        private void ExecuteAlphabetizeCommand()
+        {
+            string? text = _textBackup;
+            if (text is null)
+            {
+                return;
+            }
+
+            IOrderedEnumerable<string> lines = text.Split('\r').OrderBy(line => line, StringComparer.CurrentCulture);
+            text = string.Join('\r', lines);
+
+            lock (_lockObject)
+            {
+                _forbidBackup = true;
+                Text = text;
+                _forbidBackup = false;
+            }
+        }
+
+        #endregion
+
+        #region ReverseAlphabetizeCommand
+
+        public IRelayCommand ReverseAlphabetizeCommand { get; }
+
+        private void ExecuteReverseAlphabetizeCommand()
+        {
+            string? text = _textBackup;
+            if (text is null)
+            {
+                return;
+            }
+
+            IOrderedEnumerable<string> lines = text.Split('\r').OrderByDescending(line => line, StringComparer.CurrentCulture);
```

Comment:
Could we use `Environment.NewLine` instead of `\r` ? It's more future-proof as DevToys 2.0 will target MacOS and Linux.

Reply:
As mentioned in the original comment, the `CustomTextBox` was only giving me lines separated by `\r`, even when loading from a file with different line endings! `Environment.NewLine`, which is `\r\n` on Windows, would not work.

What I can do is split on all three variations (`\r`, `\n`, `\r\n`) to cover all cases. What do you think?

Or maybe you can check the behavior of DevToys 2.0 on another platform and see if `CustomTextBox` still has only `\r`.

Reply:
I see. Let's keep it as `\r` for now then. Will adapt this when migrating to DevToys 2.0 (FYI we will have to recode all the tools from scratch 😅)